### PR TITLE
[Bybit] handle inverse positions and associated orders creation

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: Trading/Exchange/bybit/bybit_exchange.py [hunk 1]
```diff
@@ -239,7 +243,8 @@ def _get_position_idx(self, contract):
         # 0-One-Way Mode
         # 1-Buy side of both side mode
         # 2-Sell side of both side mode
-        if contract.is_one_way_position_mode():
+        if contract.is_one_way_position_mode() or contract.position_mode is None:
```
Comment (GuillaumeDSM) [inline Trading/Exchange/bybit/bybit_exchange.py:246]: todo figure out what we do with bybit position mode unset on inverse

File: Trading/Exchange/bybit/bybit_exchange.py [hunk 2]
```diff
@@ -110,7 +112,12 @@ def get_default_type(self):
         return 'spot'
 
     async def get_positions(self) -> list:
-        return self.parse_positions(await self.connector.client.fetch_positions())
+        params = {}
+        raw_positions = []
+        for position_type in ("linear", "inverse"):
+            params["type"] = position_type
+            raw_positions += await self.connector.client.fetch_positions(params=params)
```
Comment (Herklos) [inline Trading/Exchange/bybit/bybit_exchange.py:119]: 😮

**Review by Herklos — APPROVED** [review]
Nice work 👍

File: Trading/Exchange/bybit/bybit_exchange.py (hunk 2 above)
Comment (GuillaumeDSM) [reply Trading/Exchange/bybit/bybit_exchange.py:119]: they are separated :/

**Pull request merged**